src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResource.kt

```diff
@@ -57,6 +59,14 @@ class IncentiveReviewsResource(private val incentiveReviewsService: IncentiveRev
     @PathVariable
     levelCode: String,
 
+    @Schema(description = "Sort reviews by", required = false, defaultValue = "NEXT_REVIEW_DATE", example = "NEXT_REVIEW_DATE", allowableValues = ["NEXT_REVIEW_DATE"])
+    @RequestParam(required = false)
+    sort: IncentiveReviewSort? = null,
```

Comment:
Minor one on the name, shouldn't be reviews (plural) for consistency with other things related to this reviews endpoint? (but maybe there is a reason why this is singular)

Reply:
Are you referring to the class name `IncentiveReviewSort`? I think it’s more natural English to call something that sorts reviews (plural) a "review sort" (no plural on the noun adjunct); like "card sort". The annoying side effect is, as you point out, that the class name prefix differs between classes… not sure which matters more 🤪